src/Routing/Router.php

```diff
@@ -979,7 +982,39 @@ public static function scope($path, $params = [], $callback = null)
         if (is_array($params)) {
             $options = $params;
             unset($params['routeClass'], $params['extensions']);
+            $cache = (string)Hash::get($params, '_cache');
+            if (!empty($cache)) {
+                unset($params['_cache']);
+                static::$_collection = Cache::remember(
+                    $path,
```

Comment:
Should these keys be prefixed with `routes` to avoid potential conflicts with other cached data?